test/normal_failure.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo "Subprocess started..."
+sleep 10
+echo "Subprocess ended abnormally."
+exit 1
```

Comment:
Ten seconds is *eons* in test running time, and even one second is rather longer than we want to sit around waiting. Since we know we have Python available, maybe use little Python programs as the test scripts?

```python
#!/usr/bin/env python
import sys, time
print("Subprocess started...")
time.sleep(0.2)
print("Subprocess exploded into billions of tiny shards.")
sys.exit(123)
```